src/module/item/entity.js

```diff
@@ -152,16 +152,18 @@ export default class OseItem extends Item {
     });
   }
 
-  spendSpell() {
+  spendSpell = async () => {
```

Comment:
why the break with function declaration convention established elsewhere? Did eslint call for it here?

 I thought we were only using `= async ()` for helper function exports not class members, but I could be wrong

Reply:
I think this was done in the heat of the moment while writing all the tests and such! I'll submit a new commit to fix this!